src/scripts/QuestScripts/Quest_TirisfalGlades.cpp

```diff
@@ -125,10 +125,31 @@ class Zealot : public CreatureAIScript
     }
 };
 
+class FreshOutOfTheGrave : public QuestScript
+{
+public:
+    void OnQuestStart(Player* mTarget, QuestLogEntry* /*qLogEntry*/) override
+    {
+        uint32_t rigorMortisSpell = 73523;
+        uint32_t ressurrectSpell = 73524;
+
+        // Ressurect our Player
+        mTarget->castSpell(mTarget, sSpellMgr.getSpellInfo(ressurrectSpell), true);
+
+        // Remove death Aura
+        if (mTarget->HasAura(rigorMortisSpell))
+        {
+            mTarget->removeAllAurasById(rigorMortisSpell);
+            mTarget->removeSpell(rigorMortisSpell, false, false, 0);
```

Comment:
Is removeSpell necessary here?

Reply:
its a starting spell of undeady so yeah u may want to remove it completly

Reply:
So it's in the player's spellbook? Should it be there in the first place?

Reply:
dont looks like in database its only on spell_cast so removeSpell is not necessairy here :)

Reply:
but also is wrong in my eyes to just relog and dont have the spell anymore without doing the quest

Reply:
Hmm yeah I think spells from playercreateinfo_spell_cast are only cast at first login. But then again if you logout the aura should be saved to db (though saving and loading auras from db is still legacy code :P).